src/genetic.c

```diff
@@ -622,31 +612,40 @@ geneticAlgorithm(genetic_real* bestFitValues, const unsigned int parameterCount,
                  unsigned int generationSize, unsigned int tourneySize,
                  const unsigned int maximumIterationCount,
                  fitness_evaluation_function evaluationFunction,
+                 unsigned numberOfEliteValues,
```

Comment:
This parameter should be const?

Reply:
No because i want to handle the case if itsnon pair number, I don't want to throw an error just because of that